e2e/specs/st_image_replay.spec.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+describe("st.image replay", () => {
```

Comment:
I feel like these e2e tests don't really test what we want aka that the images are cached and faster. Correct me if I'm wrong but I'm not sure if we should maybe be including these e2e tests because these just test that 2 images appear.

Reply:
While the tests could be more thorough by verifying basic caching functionality, that feels low value to me when we already have tests that caching happens. I'm only really concerned about checking if images replay, because I already trust that stuff will come from the cache.